Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -72,14 +72,10 @@
   --remotion-themed-outline-offset-focus: calc((var(--remotion-themed-border-width) + var(--remotion-themed-outline-width)) * -1);
   --remotion-themed-outline-offset-focus-hover: var(--remotion-themed-outline-offset-focus);
   --remotion-themed-outline-offset-readonly-focus: var(--remotion-themed-outline-offset-focus);
-  --remotion-themed-scrollbar-width: 10px;
-  --remotion-themed-scrollbar-border-radius: 10px;
   --remotion-themed-scrollbar-style: thin;
```

Comment:
the rename of the css property is still open. I'm assuming because the PR isn't ready yet? Or is the rename intentionally left out because the rename isn't helpful?

Reply:
yea, mainly because it wasnt ready yet